pymtl3/passes/sverilog/import_/verilator_wrapper.py.template

```diff
@@ -140,16 +144,21 @@ class {component_name}( Component ):
     def set_output():
 {set_comb_output}
 
-    @s.update_on_edge
-    def seq_upblk():
-      # Advance the clock
-      _ffi_m.clk[0] = 0
-      _ffi_inst.eval( _ffi_m )
-      _ffi_m.clk[0] = 1
-      _ffi_inst.eval( _ffi_m )
-      set_output()
-
-    {constraint_str}
+    if {has_clk}:
```

Comment:
most of the time, has_clk should be true .. right?

Reply:
Yes